openml-datarobot/src/test/java/com/feedzai/openml/datarobot/DataRobotModelProviderLoadTest.java

```diff
@@ -139,4 +145,26 @@ public void invalidModelFileFormatTest() throws IOException {
                 .as("list of errors")
                 .hasSize(1);
     }
+
+    /**
+     * Checks that it is possible to get debug information for the user in the error yielded when there is a schema
+     * mis-match in the load of a model.
+     *
+     * @since 0.5.1
+     */
+    @Test
+    public void schemaMismatchDebugInfoTest() {
+        final String modelPath = this.getClass().getResource("/" + SECOND_MODEL_FILE).getPath();
```

Comment:
we could use the getter to access SECOND_MODEL_FILE

Reply:
which getter?

Reply:
forget about this, we have a getter for FIRST_MODEL_FILE (AbstractDataRobotModelProviderLoadTest#getValidModelDirName) but not for SECOND_MODEL_FILE